admin/app/com/lucidchart/piezo/GeneratorClassLoader.scala

```diff
@@ -46,7 +47,7 @@ class GeneratorClassLoader extends ClassLoader(classOf[GeneratorClassLoader].get
 
   def getClassLoader: GeneratorClassLoader = this
 
-  def loadClass[T](name: String, clazz: Class[T]) = loadClass(name).asInstanceOf[Class[_ <: T]]
+  def loadClass[T](name: String, clazz: Class[T]): Class[_ <: T] = loadClass(name).asInstanceOf[Class[_ <: T]]
```

Comment:
Nit: Since this is Scala 3, we should use `Class[? <: T]` instead.
https://docs.scala-lang.org/scala3/reference/changed-features/wildcards.html